### PR TITLE
Fix: Fixing sparkline loading state, show it on demo and adding tests

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Fixes the loading state of the sparkline chart.
It also adds it to the demo and adds tests to line, stacked area and sparkline loading states.

## Motivation and Context

Demo wasn't showing the loading state, and it was broken.

## How Has This Been Tested?

Added tests

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/charts/sparkline.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ define(function(require){
     const d3Selection = require('d3-selection');
     const d3Transition = require('d3-transition');
 
-    const {exportChart} = require('./helpers/export');
+    const { exportChart } = require('./helpers/export');
     const colorHelper = require('./helpers/color');
-    const {line} = require('./helpers/load');
+    const { stackedArea } = require('./helpers/load');
```
Comment (DalerAsrorov) [inline src/charts/sparkline.js]: Seems a bit confusing to leave the name of the loader as `stackedArea`. I would recommend something like:
```javascript
const { stackedArea: stackedAreaLoader } = require('./helpers/load');
```
Just to make it clear that it's a loading state not the chart itself.

File: src/charts/stacked-bar.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ define(function(require){
     const assign = require('lodash.assign');
     const d3Transition = require('d3-transition');
 
-    const {exportChart} = require('./helpers/export');
+    const { exportChart } = require('./helpers/export');
     const colorHelper = require('./helpers/color');
-    const {bar} = require('./helpers/load');
+    const { bar } = require('./helpers/load');
```
Comment (DalerAsrorov) [inline src/charts/stacked-bar.js]: Maybe an overkill but probably same here too

File: src/charts/sparkline.js (hunk 1 above)
Comment (Golodhros) [reply src/charts/sparkline.js]: I was super close to do that, you just gave me the final push!
Thanks!

File: src/charts/stacked-bar.js (hunk 2 above)
Comment (Golodhros) [reply src/charts/stacked-bar.js]: +1

**Pull request merged**